src/EnergyPlus/WindowManager.hh

```diff
@@ -75,9 +74,16 @@ namespace Window {
 
     int constexpr maxGlassLayers = 5;
     int constexpr maxGapLayers = 5;
-    int constexpr maxIncidentAngles = 20;
     int constexpr maxSpectralDataElements = 800; // Maximum number in Spectral Data arrays.
 
+    int constexpr numPhis = 10;
+    Real64 constexpr dPhiDeg = 10.0;
+    Real64 constexpr dPhiRad = dPhiDeg * Constant::DegToRad;
+
+    constexpr std::array<Real64, numPhis> cosPhis =
```

Comment:
Bake `cos(phi)` at 10.0 degree increments into the executable, no need to compute at runtime.